### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by WardenRF** [ControlCompany/ControlCompany#18]
This mod is very cool, but I can see more potential for it. It would be very awesome if you added a way to randomize who becomes a monster at the start of each game. It's probably easier said than done but I think it would be possible and personally, I think it's a great idea and so far, you're the only team I've seen get close to doing this. Would be appreciated if you took this idea into thought, I think it can be fun for everyone.

**Comment by klampzlamps** [comment]
yep, agreed! I'm looking into this right now! I definitely have some ideas for this as well :)

**Comment by ZachFNF** [comment]
can u make this like a different mode and not mandatory to use the mod? just checking
